packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -0,0 +1,317 @@
+import { ListNext, ListItemNext, ListNextProps } from "@salt-ds/lab";
+
+type ItemType = { label: string; value: string };
+
+const ITEMS: ItemType[] = [
+  { label: "list item 1", value: "item 1" },
+  { label: "list item 2", value: "item 2" },
+  { label: "list item 3", value: "item 3" },
+];
+
+// Uncontrolled single select list
+describe("An uncontrolled single select list", () => {
+  const SingleSelectList = (props: ListNextProps) => {
+    return (
+      <ListNext {...props}>
+        {props.children ||
+          ITEMS.map((item, index) => {
+            return (
+              <ListItemNext value={item.value} key={index}>
+                {item.label}
+              </ListItemNext>
+            );
+          })}
+      </ListNext>
+    );
+  };
+
+  describe("List height", () => {
+    it("should render list with 3 list items", () => {
+      cy.mount(<SingleSelectList />);
+
+      cy.findByRole("listbox").then(($el) => {
+        expect($el.height()?.toFixed(0)).to.equal("108"); // 36px * 3
+      });
+    });
+
+    it("should render list with 2 list items, with displayedItemCount lower than childrenCount", () => {
```

Comment:
i believe the list will actually render all 3 items, but only 2 will be visible and there will be a scrollbar. I think the description should be amended